curvefs/src/client/rpcclient/task_excutor.cpp

```diff
@@ -176,6 +181,9 @@ void TaskExecutor::RefreshLeader() {
     task_->retryDirectly = (oldTarget != task_->target.metaServerID);
 }
 
+void TaskExecutor::OnPartitionAllocIDFail() {
+    metaCache_->MarkPartitionFull(task_->target.partitionID);
```

Comment:
should we let the full state persists to MDS ?
e.g., if we have a huge filesystem that contains hundreds of full partitions, and create inode will fail hundreds of times, and in the last, we create a new partition.

another possible solution is if create inode got three consecutive failures by `PARTITION_ALLOC_ID_FAIL`, create a new partition immediately

Reply:
yes, I have discuss with @cw123 ,  heartbeat with metaserver and mds should report this message. I mark TODO here